python/doc/theory/meta_modeling/chaos_basis.rst

```diff
@@ -3,132 +3,191 @@
 
-  where :math:`\delta_{j,k} = 1` if :math:`j=k` and 0 otherwise, and
-  the :math:`a_{j}`\ ’s are deterministic coefficients that fully
-  characterize the response :math:`\underline{Y}`.
+PC basis with independent random variables
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-| **Building of the PC basis – independent random variables**
+We first consider the case of *independent* input random variables.
+In this case, we will show how to create a orthogonal basis of
+multi-variate polynomials by using the *tensor product* of
+univariate orthogonal polynomials.
 
-| We first consider the case of *independent* input random variables. In
-  practice, the components :math:`X_i` of random vector
-  :math:`\underline{X}` are rescaled using a specific mapping
-  :math:`T_i`, usually referred to as an *isoprobabilistic
-  transformation* (see ). The set of scaled random variables reads:
+In practice, the components :math:`X_i` of random vector
+:math:`\vect{X}` are standardized using a specific mapping
+:math:`T_i`, usually referred to as an *isoprobabilistic
+transformation* (see :ref:`isoprobabilistic_transformation` for
+details on this topic).
+The set of standardized random variables reads:
 
-  .. math::
+.. math::
     :label: PC_isotransfo
 
-      U_i \, \, = \, \, T_i(X_i) \quad \quad , \quad \, i=1,\dots,n
+    Z_i  =  T_i(X_i), \quad \, i=1,\dots,n_X
 
-  Common choices for :math:`U_i` are standard distributions such as a
-  standard normal distribution or a uniform distribution over
-  :math:`[-1,1]`. For simplicity, it is assumed from now on that the
-  components of the original input random vector :math:`\underline{X}`
-  have been already scaled, i.e. :math:`X_i = U_i`.
+Common choices for :math:`Z_i` are standard distributions such as a
+standard normal distribution (which leads to the Hermite orthogonal
+polynomial basis) or a uniform distribution over :math:`[-1,1]` (which leads
+to the Legendre orthogonal polynomial basis).
 
-| Let us first notice that due to the independence of the input random
-  variables, the input joint PDF may be cast as:
+Notice that the independence of the input random
+variables implies that the input joint PDF is:
 
-  .. math::
+.. math::
     :label: 3.010qua
 
-      f_{\vect{X}}(\vect{x}) \, = \, \prod_{i=1}^{n} f_{X_i}(x_{i})
+    f_{\vect{Z}}(\vect{z}) = \prod_{i=1}^{n_X} f_{Z_i}(z_i)
 
-  where :math:`f_{X_i}(x_{i})` is the marginal PDF of :math:`X_i`. Let
-  us consider a family :math:`\{\pi^{(i)}_{j}, j \in \Nset\}` of
-  orthonormal polynomials with respect to :math:`f_{X_i}`, :
+where :math:`f_{Z_i}(z_i)` is the marginal PDF of :math:`Z_i`.
+For any input marginal index :math:`i \in \{1, ..., n_X\}`,
+let us consider a family :math:`\left\{\pi^{(i)}_j, j \in \Nset\right\}` of
+orthonormal polynomials with respect to :math:`f_{Z_i}` where
+:math:`j` represents the degree of the polynomial :math:`\pi^{(i)}_j`.
+By hypothesis, the polynomials are orthonormal, which implies:
```

Comment:
```suggestion
By hypothesis, the polynomials are orthonormal:
```
The equation that follows is not implied, it is the definition of orthonormality.